assets/dev/js/editor/editor-base.js

```diff
@@ -1531,4 +1551,49 @@ export default class EditorBase extends Marionette.Application {
 
 		$files.attr( { type } );
 	}
+
+	getElementType( elType, widgetType = false ) {
+		let index = elType,
```

Comment:
Maybe that's better?

```suggestion
		let id = elType,
```

OR

```suggestion
		let key = elType,
```